tools/test_parser_generator.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+#
+# Validate that the current version of the generated parser matches the output
+# generated by the version of goyacc installed on the local system.
+#
+# This is used in travis to verify that the currently committed version was
+# generated with the proper version of goyacc.
+
+CUR="sql.go"
+TMP="/tmp/sql.$$.go"
+
+
+if ! cd go/vt/sqlparser/
+then
+        echo "ERROR: $0 must be run in the root project directory"
+        exit 1
+fi
+
+goyacc -o $TMP sql.y
+gofmt -w $TMP
+
+if ! diff -q $CUR $TMP > /dev/null
+then
+        echo "ERROR: Regenerated parser $TMP does not match current version $(pwd)/sql.go:"
```

Comment:
Please mention here that the user has to run `make parser` and commit the changed `sql.go` file.